src/pages/help-centre/_article.tsx

```diff
@@ -9,6 +8,16 @@ import { SideTab, StyledLink } from 'components/elements'
 import { Container, SEO } from 'components/containers'
 import device from 'themes/device'
 
+export type ArticleProps = {
```

Comment:
Do we need `export` here?

Reply:
Used in later commits and forgot to delete. extra exports are deleted